hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/KeyValueStreamDataChannel.java

```diff
@@ -212,7 +219,9 @@ void assertOpen() throws IOException {
   public void close() throws IOException {
     if (closed.compareAndSet(false, true)) {
       try {
-        putBlockRequest.set(closeBuffers(buffers, super::writeFileChannel));
+        final long l = Time.monotonicNow();
```

Comment:
We should use more meaningful variable names, like start.
```suggestion
        final long start = Time.monotonicNow();
```

Reply:
Sure. Thanks @jojochuang .